app/views/wiki_pages/_form.html.erb

```diff
@@ -22,7 +22,7 @@
 
     <%= f.input :edit_reason, label: "Edit Reason" %>
 
-    <% if @wiki_page.errors[:title].present?  %>
+    <% if CurrentUser.is_janitor? && @wiki_page.errors[:title]&.any? { |error| error.include?("Move the posts and update any wikis linking to this page first.") }  %>
```

Comment:
Are there even any other errors for titles? Relying this specifically on the error message is as bad as relying on the fact that there is only one. The label for this is good enough

Reply:
Yes, providing an invalid title, which is the whole crux of this pr

![image](https://github.com/e621ng/e621ng/assets/17226394/570a454b-508f-4bee-baf5-212e2ea22238)

I had originally only tested for "Move the posts", but threw in the rest for extra credit